### DISCUSSION TITLE
Is it possible to have the "right side prompt" setup act as a notification listener?

### DISCUSSION THREAD

**Discussion by MostHated** [starship/starship#3581] [General]
Hey there,
 I was wondering if it were at all possible to have "right side" prompt setup act as a notification listener or receive and display a string asynchronously (instead of simply updating when a command is run)? It would be nice if, say I had an idle terminal open and I receive a message from something, the terminal could display it on the right side. I understand that this could certainly be out of the realm of what Starship is capable of, but I thought it would be at least worth an ask.

Thanks,
-MH

**Comment by andytom** [comment]
At the moment it is not possible, there is an open issue to investigate this but I can't seem to find it at the moment.
